STM32/Util/Src/systeminfo.c

```diff
@@ -304,4 +305,30 @@ void setBoardVoltage(float voltage){ BS.voltage = voltage; }
 void setBoardCurrent(float current){ BS.current = current; }
 void setBoardUsbError(uint32_t err) {BS.usb = err; }
 void setFirmwareBoardType(BoardType type){ BS.boardType = type; }
-void setFirmwareBoardVersion(pcbVersion version){ BS.pcb_version = version; }
\ No newline at end of file
+void setFirmwareBoardVersion(pcbVersion version){ BS.pcb_version = version; }
+
+/*!
+** @brief Sets the board type and version the firmware was compiled for
+**
+** If the board type/version does not match the value programmed in the OTP, this function sets the 
+** board status version error flag, and returns -1
+**
+** @param[in] type              Type of board, e.g. AC, DC, Current ...
```

Comment:
The board types of AC and DC are actually AC_Board and DC_Board. Not that it matters to much just thought it might be confusing.